unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -630,17 +630,26 @@ private void ProcessCatalog(BaseDictionary<string, WearableItem> catalog)
     {
         wearablesByCategory.Clear();
         view.RemoveAllWearables();
+        bool hasSkin = false;
+        bool hasCollectible = false;
         using (var iterator = catalog.Get().GetEnumerator())
         {
             while (iterator.MoveNext())
             {
                 if (iterator.Current.Value.IsEmote())
                     continue;
 
+                if (iterator.Current.Value.IsFromThirdPartyCollection 
```

Comment:
Had to add this extra check, since using the WITH_ITEMS param brings the thirdparty collection from start. If we dont mind that this is not going to work with WITH_ITEMS param, this check can be dismissed